src/applications/facility-locator/components/search-results-items/EmergencyCareResult.jsx

```diff
@@ -48,9 +47,6 @@ const EmergencyCareResult = ({ provider, query }) => {
           <a
             href="https://www.va.gov/COMMUNITYCARE/programs/veterans/Emergency-Care.asp"
             target="_/blank"
-            onClick={() => {
-              recordEvent({ event: 'cta-emergency-benefit-button-click' });
```

Comment:
This event is duplicative and the event from the `va-alert` is sufficient for our purposes.

<img width="600" alt="Screenshot 2024-08-08 at 3 21 39 PM" src="https://github.com/user-attachments/assets/e46d642c-e3e3-4679-a4a7-337d2d63db3f">